### ISSUE TITLE
Testers needed to test http://kartikm.github.io/fonts-aakar/

### ISSUE THREAD

**Issue by samyakbhuta** [kartikm/fonts-aakar#11]
To browse and check usability of http://kartikm.github.io/fonts-aakar/ on device of your choice.
- [x] Desktop 
- [x] iPad
- [ ] iPhone
- [ ] Android Phones
- [ ] Android Tablets

Add your device to the list if not present.

**Comment by samyakbhuta** [comment]
@kartikm, @konarak, @nehaldattani shoutout

**Comment by samyakbhuta** [comment]
Desktop ( OSX-Chrome ) tested by Samyak. OK.
Desktop ( OSX-Firefox ) tested by Samyak. OK.
Desktop ( OSX-Safari ) tested by Samyak. OK.

**Comment by samyakbhuta** [comment]
iPad ( Safari ) tested by Samyak. OK.
iPad ( Chrome ) tested by Samyak. OK.

**Comment by samyakbhuta** [comment]
For everybody testing on mobile, can you please post a screenshot ? I want to see how it looks as well. Thanks in advance.

**Comment by samyakbhuta** [comment]
http://kartikm.github.io/fonts-aakar/ has a update now.

#### New things to test
- Sample text
- Font name in Gujarati

This also means that we are downloading web font from web. It is slowing down page rendering time for sure. Please let me know if the waiting time is within bearable limit or it is frustrating ? Thanks.

**Comment by samyakbhuta** [comment]
I found following issues. Listing down below.
- આકાર - 'આ' is not rendered properly. The kerning ( space between) of _kanas_ looks less then proper. Individual [issue](https://github.com/kartikm/fonts-aakar/issues/17)
- Some glyph appear thinner then other. See 'ર્ય'  of 'ઐશ્વર્યવાન'. The _Ardh Chndrakar_ looks thinner. Individual [issue](https://github.com/kartikm/fonts-aakar/issues/16)
- ઋષી - 'ષી' is totally gone awry. Individual [issue](https://github.com/kartikm/fonts-aakar/issues/5).
- White space when glyph intersects. See 'ર્ય','ગ્રે','સ','ભે','ષી','પ્ર','જ્ઞા' and 'ભ'. Might have to zoom into the image to see it. Individual [issue](https://github.com/kartikm/fonts-aakar/issues/15).

iPad screenshot :
![img_0881](https://f.cloud.github.com/assets/738746/1664781/24daf8ae-5c2c-11e3-8c44-42523d673147.PNG)

**Comment by kartikm** [comment]
ષ is known issue #5

**Comment by samyakbhuta** [comment]
Yep. Updated a while ago. Guess time is soon that we have to open font
forge. Must try to 'lohitize' as well.

**Comment by kartikm** [comment]
#1 is still major blocker to do anything.

**Comment by samyakbhuta** [comment]
How do we bust #1 ? anything in your mind ?

**Comment by samyakbhuta** [comment]
Please also see testing on Samsung Galaxy Tab - https://github.com/kartikm/fonts-aakar/issues/24
